src/BasicPage.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Header from './Header';
+import Content from './Content';
+import Footer from './Footer';
+
+export default class BasicPage extends React.Component {
+  constructor() {
```

Comment:
constructor is not required every time and can be skipped if no initial state setup or initial config is required